tests/pytests/test_pyside_cubegen.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
i've got this remedied in a different PR, but for now, you'll have to register this file, so it gets copied over into the runtime tests https://github.com/psi4/psi4/blob/master/psi4/CMakeLists.txt#L265

Reply:
Done.